src/database/mock.rs

```diff
@@ -361,30 +357,36 @@ impl OpenTransaction {
     }
 
     fn commit(&mut self, db_backend: DbBackend) -> bool {
-        if self.transaction_depth == 0 {
-            self.push(Statement::from_string(db_backend, "COMMIT".to_owned()));
-            true
-        } else {
-            self.push(Statement::from_string(
-                db_backend,
-                format!("RELEASE SAVEPOINT savepoint_{}", self.transaction_depth),
-            ));
-            self.transaction_depth -= 1;
-            false
+        match self.transaction_depth {
+            0 => {
```

Comment:
One more indentation is also not necessary

Reply:
I reverted it https://github.com/SeaQL/sea-orm/pull/1231/commits/840cd543b002b0f6578093d76a08cb1b47b23433